dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/AppsHelper.java

```diff
@@ -426,12 +634,26 @@ private AppSecrets protectHiddenSecrets(final AppSecrets appSecrets){
         final Map<String,Secret> sourceSecrets = appSecrets.getSecrets();
         for (final Entry<String, Secret> secretEntry : sourceSecrets.entrySet()) {
             if(secretEntry.getValue().isHidden()){
-                builder.withHiddenSecret(secretEntry.getKey(), PROTECTED_HIDDEN_SECRET);
+                builder.withHiddenSecret(secretEntry.getKey(), HIDDEN_SECRET_MASK);
             } else {
                 builder.withSecret(secretEntry.getKey(),secretEntry.getValue());
             }
         }
         return builder.build();
     }
 
+    /**
+     * This method checks if we're looking at a char array all filled with the character `*`
+     * @param chars
+     * @return
+     */
+    private boolean isAllFilledWithAsters(final char [] chars){
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5329810)